vscode-wpilib/src/cpp/simulatewindows.ts

```diff
@@ -37,7 +37,7 @@ export async function startWindowsSimulation(commands: IWindowsSimulateCommands)
       name: 'HALSIM_EXTENSIONS',
       value: commands.extensions,
     }],
-    externalConsole: true,
+    console: 'integratedTerminal',
     name: 'WPILib C++ Simulate',
```

Comment:
From one issue I found…
> Use "console": "externalTerminal" if you wanted externalConsole: true or console: "internalConsole" if you wanted externalConsole: false

The PR switches everything to internalConsole.  Is that what we want?

Reply:
internal console is fine now. Vs code supports multiple consoles easily. It didn't do them right when we originally did this plugin.